pages/clubs/ClubForm.jsx

```diff
@@ -60,7 +76,18 @@ var ClubForm = React.createClass({
               <h2>{ username ? this.state.headings[this.state.currentStep] : this.state.loginHeading }</h2>
             </Illustration>
           </section>
-          { this.state.loggedIn ? this.renderSteps() : this.renderLoginRequest() }
+
+          <div hidden={!this.state.loggedIn}>
+            <Form onUpdate={this.checkAnswer} fields={{studentData: studentData}}></Form>
+
+            <div hidden={!this.state.isInterestedStudent} className="navigation">
+              <a href="https://campus.mozilla.community/" target="_blank" className="button">Go to Mozilla Campus Club Page</a>
+            </div>
+          </div>
+
+          { this.state.isInterestedStudent === false && this.renderSteps() }
+
+          { !this.state.loggedIn && this.renderLoginRequest() }
```

Comment:
Can we make this side-effect free code again? I know this cide works, but it explicitly bakes in "making things happen as a side effect" and that's a good way to get future bugs. 

I'd strongly recommend turning this into `{ !this.state.loggedIn ? this.renderLoginRequest() : null }` so that it's clear what we want to see happening, when, rather than relying on a side-effect *not* triggering due to a logical short-circuit.